mesheryctl/internal/cli/root/perf/apply.go

```diff
@@ -261,14 +278,45 @@ var applyCmd = &cobra.Command{
 	},
 }
 
+func multipleProfileConfirmation(profiles []models.PerformanceProfile) int {
+	reader := bufio.NewReader(os.Stdin)
+
+	for index, a := range profiles {
+		fmt.Printf("Index: %v\n", index)
+		fmt.Printf("Name: %v\n", a.Name)
+		fmt.Printf("ID: %s\n", a.ID.String())
+		fmt.Printf("Endpoint: %v\n", a.Endpoints[0])
+		fmt.Printf("Load Generators: %v\n", a.LoadGenerators[0])
+		fmt.Println("---------------------")
+	}
+
+	for {
+		fmt.Printf("Enter the index of profile?? ")
+		response, err := reader.ReadString('\n')
+		if err != nil {
+			log.Fatal(err)
+		}
+		response = strings.ToLower(strings.TrimSpace(response))
+		index, err := strconv.Atoi(response)
+		if err != nil {
+			log.Info(err)
+		}
+		if index < 0 || index >= len(profiles) {
+			log.Info("Invalid index")
+		} else {
+			return index
+		}
+	}
+}
+
 func init() {
-	applyCmd.Flags().StringVar(&testURL, "url", "", "(required) Endpoint URL to test")
+	applyCmd.Flags().StringVar(&testURL, "url", "", "(required/optional) Endpoint URL to test")
 	applyCmd.Flags().StringVar(&testName, "name", "", "(optional) Name of the Test")
 	applyCmd.Flags().StringVar(&profileName, "profile", "", "(required/optional) Name for the new Performance Profile")
 	applyCmd.Flags().StringVar(&testMesh, "mesh", "None", "(optional) Name of the Service Mesh")
 	applyCmd.Flags().StringVar(&qps, "qps", "0", "(optional) Queries per second")
 	applyCmd.Flags().StringVar(&concurrentRequests, "concurrent-requests", "1", "(optional) Number of Parallel Requests")
-	applyCmd.Flags().StringVar(&testDuration, "duration", "30s", "(optional) Length of test (e.g. 10s, 5m, 2h). For more, see https://golang.org/pkg/time/#ParseDuration")
+	applyCmd.Flags().StringVar(&testDuration, "duration", "30s", "(optional) Length of test (e.g. 10s, 5m, 2h). For more, see https://golang.org/pkg/time/-ParseDuration")
```

Comment:
Looks like the url is not found
<img width="915" alt="image" src="https://user-images.githubusercontent.com/53065463/122980597-c0e64580-d390-11eb-8eec-6a88843bcdb3.png">

Reply:
Oops I'll correct that!